src/core/lib/transport/call_filters.h

```diff
@@ -43,6 +43,7 @@
 // - OnServerInitialMetadata  - $VALUE_TYPE = ServerMetadata
 // - OnServerToClientMessage  - $VALUE_TYPE = Message
 // - OnClientToServerMessage  - $VALUE_TYPE = Message
+// - OnClientToServerHalfClose - no value
```

Comment:
I thought we were just going to call this `OnClientHalfClose`?  That's what we put in the internal doc we wrote for estubs.